### ISSUE TITLE
Crashes with unstable internet

### ISSUE THREAD

**Issue by Stereowalker** [adlerluiz/ytmdesktop#1393]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

Whenever your internet is very slow and a song is playing, the app wil crash a short while later

### Expected Behavior

The song to continue buffering the song without crashing

### Steps To Reproduce

You'll just need relatively bad internet, something that peaks at around 100kbps. Open the app and playing any song should crash it

### YTMDesktop

v2.0.5

### OS

Linux (Ubuntu)

### OS Version

6.5.0-41

### Arch

x64

### Installation way

.deb

### Anything else?

YouTube Music Desktop App Crashed

Environment Details:
    YouTube Music Desktop App 2.0.5
    Electron 29.0.1
    Linux 6.5.0-41-generic

Name: AggregateError
Message: 
Cause: Unknown

AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at internalConnectMultiple (node:net:1177:5)
    at Timeout.internalConnectMultipleTimeout (node:net:1687:3)
    at listOnTimeout (node:internal/timers:575:11)
    at process.processTimers (node:internal/timers:514:7)

**Comment by NovusTheory** [comment]
Possibly related to other issues pertaining to networking as this error can't be reproduced by us easily and it originates from node with no stack trace to the app so it doesn't help us in any way.

Will try an artificial network throttle to see if I can reproduce it.

**Comment by Akiisqt** [comment]
I was able to limit the connection to the app to under 50 kbps, I wasn't able to recreate this issue on v2.0.5, windows 11 x64. Besides the app was never actually reaching that high of usage.

**Comment by Stereowalker** [comment]
How do I go about limiting my connection? I use 2 networks, a relatively slower unlimited one that seems to be oddly connected to a VPN and a faster metered one. Maybe the crash is limited to the peculiarities of the first network, so maybe I can try to slow down my second connection

**Comment by Akiisqt** [comment]
Using software

**Comment by dipras** [comment]
Yes i had this error. This only happen when internet is slow, but not when happen when internet is down. So it's hard to produce this behavior

**Comment by Akiisqt** [comment]
Posts says nothing about lost of connection, artificially slowing my internet down did nothing to cause a crash

**Comment by dipras** [comment]
> Posts says nothing about lost of connection, artificially slowing my internet down did nothing to cause a crash

This error is really random. I was even tried to slowing my internet and the error not come up. And suddenly, just now. The error come up
![image](https://github.com/user-attachments/assets/609f1447-cf35-43e7-9cfe-463d32a071a1)

Here is the detail
```
YouTube Music Desktop App Crashed

Environment Details:
    YouTube Music Desktop App 2.0.5
    Electron 29.0.1
    Linux 6.8.0-40-generic

Name: AggregateError
Message: 
Cause: Unknown

AggregateError
    at internalConnectMultiple (node:net:1114:18)
    at internalConnectMultiple (node:net:1177:5)
    at Timeout.internalConnectMultipleTimeout (node:net:1687:3)
    at listOnTimeout (node:internal/timers:575:11)
    at process.processTimers (node:internal/timers:514:7)
```

**Comment by dipras** [comment]
My system specification is

```
OS: Kubuntu 24.04 LTS x86_64 
Host: Aspire A514-54G V1.26 
Kernel: 6.8.0-40-generic 
Uptime: 15 hours, 14 mins 
Packages: 2914 (dpkg), 18 (snap) 
Shell: zsh 5.9 
Resolution: 1920x1080, 2560x1440 
DE: Plasma 5.27.11 
WM: KWin 
Theme: [Plasma], Breeze [GTK2/3] 
Icons: [Plasma], breeze-dark [GTK2/3] 
Terminal: konsole 
Terminal Font: 0xProto Nerd Font 10 
CPU: 11th Gen Intel i7-1165G7 (8) @ 4.700GHz 
GPU: NVIDIA GeForce MX350 
GPU: Intel TigerLake-LP GT2 [Iris Xe Graphics] 
Memory: 8021MiB / 15770MiB 
```

**Comment by Alipoodle** [comment]
This should be addressed in ead0bbd58d52fa7e3f4cf060fc7aacedccb94459
